MiniApp/Classes/MiniApp.swift

```diff
@@ -95,4 +94,50 @@ public class MiniApp: NSObject {
     public func create(appInfo: MiniAppInfo, completionHandler: @escaping (Result<MiniAppDisplayProtocol, Error>) -> Void, messageInterface: MiniAppMessageDelegate) {
         return realMiniApp.createMiniApp(appInfo: appInfo, completionHandler: completionHandler, messageInterface: messageInterface)
     }
+
+    @available(*, deprecated,
+    message:"Use MASDKError instead of Error in your completionHandler.",
+    renamed: "list(completionHandler:)")
+    public func list<T>(completionHandler: @escaping (Result<[MiniAppInfo], T>) -> Void) where T: Error {
```

Comment:
I guess this is added by mistake?

@availble syntax indicates that the method is deprecated but we weren't using 
`public func list<T>(completionHandler: @escaping (Result<[MiniAppInfo], T>) -> Void) where T: Error {` before.

The method which we currently support 
`public func list(completionHandler: @escaping (Result<[MiniAppInfo], Error>) -> Void) {`

Reply:
This one is a little weird, but it was the only way I could find to support `MASDKError` as the new default but not break support of `Error` if the host app has explicitly defined that type.